Assets/HoloToolkit/UX/Scripts/Receivers/InteractionReceiver.cs

```diff
@@ -0,0 +1,343 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System.Collections.Generic;
+using HoloToolkit.Unity.InputModule;
+
+namespace HoloToolkit.Unity.Receivers
+{
+    /// <summary>
+    /// An interaction receiver is simply a component that attached to a list of interactible objects and does something
+    /// based on events from those interactible objects.  This is the base abstact class to extend from.
+    /// </summary>
+    public abstract class InteractionReceiver : MonoBehaviour
+    {
+        #region Public Members
+        /// <summary>
+        /// List of linked interactible objects to receive events for
+        /// </summary>
+        [Tooltip("Target Interactible Object to receive events for")]
+        public List<GameObject> Interactibles = new List<GameObject>();
+
+        /// <summary>
+        /// List of linked targets that the receiver affects
+        /// </summary>
+        [Tooltip("Targets for the receiver to ")]
+        public List<GameObject> Targets = new List<GameObject>();
+
+        /// <summary>
+        /// Flag for locking focus while selected
+        /// </summary>
+        [Tooltip("If true, this object will remain the prime focus while select is held")]
+        public bool bLockFocus;
```

Comment:
`BlockFocus` or `LockFocus`?

Reply:
LockFocus, b = bool.  Changed it to _LockFocus.

Reply:
Public fields should not have underscore prefix.  Why can't it be `IsLockFocused`?